test/e2e/nix/build_test.go

```diff
@@ -0,0 +1,55 @@
+package nix_test
+
+import (
+	"context"
+	"github.com/benchkram/bob/bob"
+	"github.com/benchkram/errz"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"os"
+)
+
+var _ = Describe("Testing new nix implementation", func() {
+	BeforeEach(func() {
+		Expect(os.Setenv("PATH", initialPath)).NotTo(HaveOccurred())
```

Comment:
Can probably be removed once the environment is passed on tasklevel.

Reply:
No sure, because each subtest will tamper with the PATH, so that's why I added this to make sure that before a new subtest, we have the the PATH of the user who runs the test.

Reply:
As you don't alter the path internaly anymore i guess now it's no longer needed?

Reply:
True, will remove